### PR TITLE
Added new patch release of lens 1.0.1

### PR DESCRIPTION
**Pull request by pdonadeo**

### CODE REVIEW

File: packages/lens/lens.1.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "lens"
+version: "1.0.1"
```
Comment (dsheets) [inline packages/lens/lens.1.0.1/opam]: The opam metadata files in this repository typically don't include `name` and `version` fields as those values are implied from the directory structure. Not including them makes it easier to copy and compare opam files for different versions of the same package.

File: packages/lens/lens.1.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "lens"]
+]
+depends: [
+  "ocamlfind"
```
Comment (dsheets) [inline packages/lens/lens.1.0.1/opam]: This could be marked as a `{ build }` dependency.

Comment (Chris00) [inline packages/lens/lens.1.0.1/opam]: You may want to use [oasis2opam](https://github.com/ocaml/oasis2opam) which will do that — and more — for you (using the Git version at the moment but I could make a release).  In order to have the (new version) of `ocaml-version: >= "4.01.0"`, you need to add that constraint to `_oasis`.

File: packages/lens/lens.1.0.1/opam (hunk 1 above)
Comment (pdonadeo) [inline packages/lens/lens.1.0.1/opam]: @dsheets ok, I'm fixing all the issues geneated by this patch.

**Pull request closed without merging**